flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/operations/SqlDdlToOperationConverterTest.java

```diff
@@ -1361,16 +1374,36 @@ public void testAlterTableDropConstraint() throws Exception {
                                 .getUnresolvedSchema()
                                 .getPrimaryKey())
                 .isNotPresent();
+    }
+
+    @Test
+    public void testAlterTableDropDistribution() throws Exception {
+        prepareNonManagedTableWithDistribution("tb1");
+        String expectedSummaryString = "ALTER TABLE cat1.db1.tb1\n  DROP DISTRIBUTION";
 
-        operation = parse("alter table tb1 drop primary key");
+        Operation operation = parse("alter table tb1 drop distribution");
         assertThat(operation).isInstanceOf(AlterTableChangeOperation.class);
         assertThat(operation.asSummaryString()).isEqualTo(expectedSummaryString);
-        assertThat(
-                        ((AlterTableChangeOperation) operation)
-                                .getNewTable()
-                                .getUnresolvedSchema()
-                                .getPrimaryKey())
+        assertThat(((AlterTableChangeOperation) operation).getNewTable().getDistribution())
                 .isNotPresent();
+
+        prepareNonManagedTableWithDistribution("tb3");
+        // rename column used as distribution key
```

Comment:
comment seems to be wrong, also isn't this tested above already, at least `is used as a distribution key` occurs two times in this clas

Reply:
Removing this case of dropping the distribution.  The other two cases are for a drop and modify, so I think both of those make sense.